apps/metabase/0.52.2/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   metabase:
-    image: metabase/metabase:v0.52.2
+    image: metabase/metabase:v0.52.3
     container_name: ${CONTAINER_NAME}
     ports:
       - ${PANEL_APP_PORT_HTTP}:3000
```

Comment:
The provided difference looks straightforward and has no apparent errors. The service definition for Metabase is correctly updated to use the newer version `v0.52.3`. There's nothing here that requires modification beyond updating the image tag. If you intended to make any other changes, such as changing the container name or adjusting ports, those should be reflected in this patch file or additional commits. However, based solely on what's shown, the update appears to be correct and will work without needing further changes.